modules/virt-copying-the-ssh-command.adoc

```diff
@@ -4,12 +4,15 @@
 
 :_content-type: PROCEDURE
 [id="virt-copying-the-ssh-command_{context}"]
-= Copying the SSH command from the web console
+= Copying the SSH command using the web console
+
+Copy the command to connect to a virtual machine (VM) terminal via SSH. The web console allows you to quickly access the VM terminal by using `virtctl`.
```

Comment:
> The web console allows you to quickly access the VM terminal by using `virtctl`.

I think this sentence is ambiguous. The confusing part for me is that virtctl is usable without the web console (of course) and this sentence is phrased in a way that makes it sound like _using_ virtctl is dependent on the web console - or that virtctl is something that you use within the web console rather than the CLI.

Reply:
I can see how that sentence can be confusing. I will get rid of it.